release.config.cjs

```diff
@@ -0,0 +1,83 @@
+const transform = require("./release.utils.cjs");
+module.exports = {
+    branches: [
+        "+([0-9])?(.{+([0-9]),x}).x",
+        "master",
+        "next",
+        "next-major",
```

Comment:
Any commits to the branches `next`, `next-major` will append a prelease version, like `0.0.11-beta`